### ISSUE TITLE
Formatting of multiple include directives

### ISSUE THREAD

**Issue by simsrw73** [prettier/plugin-pug#18]
This may be ignorance or misconfiguration on my part, but I'm having trouble figuring it out. 

Whenever I use two or more include statements consecutively, prettier removes all indentation after the first instance.

Eg. If I create a new project:
```
npm init -y
npm i -D pug prettier @prettier/plugin-pug
```

create an index.pug:
```
doctype 
html
    head
    body
        include in1
        include in2
        include in3
```

then run prettier: `npx prettier index.pug`

I get the output:
```
doctype
html
    head
    body
        include in1
include in2
include in3
```

What am I doing wrong? Thanks.

**Comment by Shinigami92** [comment]
I will investigate time in the next hours/days

**Comment by Shinigami92** [comment]
Released as [1.0.1](https://www.npmjs.com/package/@prettier/plugin-pug/v/1.0.1)
